lib/perl/Genome/Model/Build.pm

```diff
@@ -1868,8 +1868,8 @@ sub generate_send_and_save_report {
     my $email_confirmation = Genome::Report::Email->send_report(
         report => $report,
         to => $to->id,
-        from => 'apipe@'.Genome::Config::domain(),
-        replyto => 'noreply@'.Genome::Config::domain(),
+        from => $ENV{GENOME_EMAIL_NOREPLY},
```

Comment:
`GENOME_EMAIL_FROM` does sound more correct. That would have to be a new environment variable that we would have to define. I'm also not sure what value to set that variable to. `apipe@genome.wustl.edu` doesn't seem appropriate for this and hence I decided to stick with the `GENOME_EMAIL_NOREPLY`